lib-android/src/main/res/values/colors.xml

```diff
@@ -4,4 +4,9 @@
   <color name="lib_transparent">#00000000</color>
   <color name="lib_white">#FFFFFFFF</color>
   <color name="lib_black">#FF000000</color>
+
+  <!-- SignInButton -->
+  <color name="lib_sign_in_google">#FFE73F25</color>
```

Comment:
To discuss - what do you think about moving these specific values to other resource file with _singInButton suffix just like with the style?